dapp-test/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "dapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "serve": "python3 -m http.server 8000"
```

Comment:
consider using https://www.npmjs.com/package/serve to restraint tech stack to nodeJS

Reply:
switched to http-serve to only allow js stack